### DISCUSSION TITLE
All search buttons are invalid

### DISCUSSION THREAD

**Discussion by twgh** [community/community#110918] [Mobile] [answered]
Android client clicks search button, no effect.

Android 14.

**Comment by alcere** [comment]
H @twgh i, could you please provide some steps to reproduce the issue? Is this happening for a particular screen or across the entire app? Thank you.

**Reply by twgh** [reply]
![Screenshot_2024_0305_224555.gif](https://github.com/community/community/assets/54952145/14c28578-26d6-4b5d-8e8b-9795008d327a)

**Reply by twgh** [reply]
My phone doesn't have Google services.
Does this search button rely on Google services?

**Reply by alcere** [reply]
Thanks for the details @twgh! Have you ever used search in the past correctly? Which app store did you use to download the app?. I'm going to open an internal issue to investigate the problem. Thank you!

**Reply by twgh** [reply, marked as answer]
> Thanks for the details @twgh! Have you ever used search in the past correctly? Which app store did you use to download the app?. I'm going to open an internal issue to investigate the problem. Thank you!

After updating to the latest version, the issue has been resolved.
